docs/languages/julia.md

```diff
@@ -0,0 +1,71 @@
+# Julia in Visual Studio Code
+
+The Visual Studio Code extension for Julia includes built-in dynamic autocompletion, inline results, plot pane, integrated REPL, variable view, code navigation, and many other advanced language features.
+
+![Julia VS Code Overview](images/julia/overview.png)
+
+Most of these features just work out of the box, while some may require basic configuration to get the best experience. This page summarizes the Julia features that the Julia VS Code extension ships with. For a more in-depth guide on how these features work and can be configured, see [The Julia VS Code Docs](https://www.julia-vscode.org/docs/stable/).
+
+## Getting started
+
+1. Install Julia for your platform: https://julialang.org/downloads/
+2. Install VS Code for your platform: https://code.visualstudio.com/download
+    At the end of this step you should be able to start VS Code.
+3. Open the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=julialang.language-julia) and press "Install"; or manually install by doing the following:
+    1. Start VS Code.
+    2. Inside VS Code, go to the extensions view by
+        clicking `View` on the top tool bar and then selecting `Extensions`.
+    3. In the extensions view, simply search for the term ``julia`` in the marketplace
+        search box, then select the extension named ``Julia`` (as shown below) and click the install button.
+        You should restart VS Code after this step. Note that if you run into any issues installing the Julia VS Code Extension, check out the [docs on installing an extension](https://code.visualstudio.com/docs/editor/extension-marketplace) which should help clarify any issues.
+        
+    ![Julia in the VS Code Marketplace](images/julia/julia-extension-marketplace.png)
+
+## Running Code
+
+There are a variety of different ways to run Julia code through the VS Code IDE. You have the option to run a Julia file (via `(Ctrl+F5)`, which will run whatever Julia file you have open and active), execute Julia commands via the REPL, or even just execute a specific block of code from a file you have open. To learn more about these options, head to the [The Julia VS Code Docs section on running code](https://www.julia-vscode.org/docs/stable/userguide/runningcode/).
+
+## Debugging 
+
+You can start debugging by opening the Julia file which you would like to debug. Then, click the debug icon on the left (as shown below):
+
+![Getting started debugging Julia code](images/julia/debug1.png)  
```

Comment:
This image and the next few show a three circle icon in the Activity bar.
Is this for a view from the Julia extension? If so, should this be described somewhere in the topic?

Reply:
Also the screenshots show 5K pending changes, which might be distracting.

Reply:
Will update screenshots without the source control button active

Reply:
yes, that's a good call